roboteam_ai/src/jsons/tactics/QualificationTactic.json

```diff
@@ -48,7 +48,7 @@
               "ROLE": "qualRole",
```

Comment:
This will make the robots in the demo field cry, maybe dont push this

Reply:
fixed :p